packages/partyrpc/src/server/create-fetch-handler.ts

```diff
@@ -86,11 +86,11 @@ export function createFetchHandler<const TEndpoints extends readonly AnyEndpoint
 
 export interface PartyFetchHandler<TEndpoints extends readonly AnyEndpointDefinition[], UserContext> {
   onFetch: (
-    req: Request,
-    lobby: { env: Record<string, unknown>; parties: PartyKitRoom["parties"] },
+    req: PartyKitRequest,
+    lobby: Party.FetchLobby,
     ctx: ExecutionContext,
     userCtx: UserContext,
-  ) => UserDefinedResponse | Promise<UserDefinedResponse>;
+  ) => Response | Promise<Response>;
```

Comment:
Quite a few versions ago it seems Partykit now only supports returning Fetch API Responses from `onFetch`, removing the need for `UserDefinedResponse` .